test.js

```diff
@@ -16,6 +16,13 @@ describe('markdown-it-implicit-figures', function() {
     assert.equal(res, expected);
   });
 
+  it('should add <figure> when image is by itself in a paragraph or preceeded by a standalone link', function () {
+    md = Md().use(implicitFigures, { dataType: true, figcaption: true });
+    var src = 'text with ![](img.png)\n\n![](fig.png)\n\nanother paragraph\n\n[![Caption](fig.png)](http://example.com)\n\nyet another paragraph';
```

Comment:
I could shorten the url link (to url.html), but I inserted the caption as the code that places figcaption was modified, so wanted to make sure it places the tags in the right spot. I guess it's too late now that you merged :)

Thanks for the quick merge!